a/ansible_event-driven-ansible/build_info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "event-driven-ansible",
```

Comment:
here it should be ansible-event-driven-ansible

Reply:
done